src/pages/Chat/Chat.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState} from 'react';
+import styles from './Chat.module.css';
+import UserSearch from '../../components/UserSearch/UserSearch'
+import ChatList from '../../components/ChatList/ChatList'
+import ChatHeader from '../../components/ChatHeader/ChatHeader'
+import DialogueList from '../../components/DialoguesList/DialoguesList'
+import ChatInput from '../../components/ChatInput/ChatInput'
+import ChatBuddyDetail from '../../components/ChatBuddyDetail/ChatBuddyDetail'
+import { getBuddyChatList} from '../../api/CommunicationAPI';
+
+const mockMessage = [
+  {
+    sender: 'id3333',
+    text: 'random message',
+    contentType: 'text',
+  },
+  {
+    sender: 'id5678',
+    text: 'https://play-lh.googleusercontent.com/zepWGavYYErAIBXFZb6OT14I6b-m4TyaG3yjqZy6Hnsmi64vL3upQ3KUsV6Wnsm-e9M=w512',
+    contentType: 'image',
+  },
+];
+
+const mockCourse = [
+  {
+    courseName: 'SOFTENG 701',
+    courseLink: 'https://random.link',
+  },
+];
+
+function Chat() {
+  const [chatList,setChatList] = useState([]);
+
+  useEffect(()=>{
+       const getChatList = async () => {
+         try {const response = await getBuddyChatList();
```

Comment:
Very finnicky point, but if you get a chance, I think this is meant to be reformated?

Reply:
I'll run it through prettier later :)